cdap-docs/_common/_source/index.rst

```diff
@@ -14,20 +14,20 @@ CDAP Documentation v\ |version|
 
 .. rubric:: Introduction to the Cask Data Application Platform
 
-The Cask |(TM)| Data Application Platform (CDAP) is an open source application development
+The Cask |(TM)| Data Application Platform (CDAP) is an integrated, open source application development
```

Comment:
We took it from Boyd's pitch. "Integration - Capabilities over Features"
But we want to explain why it is integrated. Perhaps we can add a bullet in the next list:
- Integration of the various components of the Hadoop ecosystem into a single platform.